api/src/main/java/org/openmrs/module/fhir2/api/translators/TaskTranslator.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators;
+
+import org.hl7.fhir.r4.model.Task;
+
+public interface TaskTranslator extends OpenmrsFhirTranslator<org.openmrs.module.fhir2.Task, org.hl7.fhir.r4.model.Task> {
+	
+	Task toFhirResource(org.openmrs.module.fhir2.Task openmrsTask);
```

Comment:
It would be nice to have some JavaDoc remarks on these two methods. See, e.g., [the PatientTranslator](https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/PatientTranslator.java).